en/firefox/new/desktop.ftl

```diff
@@ -15,6 +15,11 @@ firefox-desktop-download-meta-desc-v2 = Get { -brand-name-firefox } for { -brand
 firefox-desktop-download-og-title = Download the fastest { -brand-name-firefox } ever
 firefox-desktop-download-og-desc = Faster page loading, less memory usage and packed with features, the new { -brand-name-firefox } is here.
 firefox-desktop-download-firefox = { -brand-name-firefox-browser }
+
+# Variables:
+#   $update_url (url) - link to https://support.mozilla.org/kb/update-firefox-latest-release
+firefox-desktop-out-of-date = An even newer { -brand-name-firefox } is available. <a { $update_url }">Update to the latest version</a>
```

Comment:
```suggestion
firefox-desktop-out-of-date = An even newer { -brand-name-firefox } is available. <a { $update_url }>Update to the latest version</a>
```

Based on the code I think that quote mark is superfluous https://github.com/mozilla/bedrock/blob/53a26385e8662c9fcb16abfe049293f33556e2a9/bedrock/firefox/templates/firefox/new/desktop/download.html#L75

Reply:
It's happened at least a couple of times (causing a cascade of weird side effects on Smartling), I wonder if it's something that can be caught by linters.

Reply:
Good question. Seems like it should be.